models/migrations/v101.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import "github.com/go-xorm/xorm"
+
+func addWhitelistDeployKeysToBranches(x *xorm.Engine) error {
+	type ProtectedBranch struct {
+		ID                  int64
+		WhitelistDeployKeys bool
```

Comment:
Shouldn't this be `NOT NULL DEFAULT false`? Otherwise we will have three possible values: `NULL`, `fa;se`, `true`.

Reply:
Done.

Reply:
Didn't have that originally because I wanted to avoid a migration.

Reply:
Well, I thought the same as you. Why migrate when xorm does an excellent work at it? But then I think it was lafriks that explained to me: later migration steps (i.e. > 101) will run before xorm does its job, so if any of those future migrations need this column, they will fail.